crates/nu-glob/src/lib.rs

```diff
@@ -965,8 +969,9 @@ mod test {
                 .ok()
                 .and_then(|p| match p.components().next().unwrap() {
                     Component::Prefix(prefix_component) => {
-                        let path = Path::new(prefix_component.as_os_str()).join("*");
-                        Some(path)
+                        let path = Path::new(prefix_component.as_os_str());
+                        path.join("*");
```

Comment:
Is this change intentional? From what I can tell line 973 isn't doing anything.

Reply:
Yeah, I just used the same code for https://github.com/rust-lang-nursery/glob/pull/105.
Quite interesting that all CI passed :thinking: